conf/messages

```diff
@@ -794,6 +798,8 @@ viewAllTemplates.noTemplates.p2 = You can create a template by submitting a draf
 viewAllTemplates.createANewMovement = Create a new movement
 viewAllTemplates.renamedNotification.title = Success
 viewAllTemplates.renamedNotification.content = Draft template "{0}" renamed to "{1}".
+viewAllTemplates.deletedNotification.title = Success
+viewAllTemplates.deletedNotification.content = Template ''{0}'' deleted.
```

Comment:
Screenshot in jira shows this message with single quotes and with word Draft at the beginning.

```suggestion
viewAllTemplates.deletedNotification.content = Draft template '{0}' deleted.
```

Reply:
Change the quotes but the copy-deck doesn't say to start with the word draft
> Template '[templateName]' deleted.

Reply:
https://github.com/hmrc/emcs-tfe-frontend/pull/241/commits/8b72bcc5b34e5fdc1ef60ccb79bd70a276573166 for the fix to the quotes on the flash message